pkg/soperator/soperator_components.go

```diff
@@ -103,7 +103,6 @@ func (specConfig *SpecConfig) GetComponents() (*api.ComponentList, error) {
 	}
 	components := &api.ComponentList{
 		Deployments:     []*components.Deployment{deployment},
-		Services:        specConfig.getOperatorService(),
```

Comment:
What is this for?

Reply:
For prometheus metrics, we created this service previously. Since we are removing prometheus metrics, we don't need it.

Reply:
This seems to have the operator service and the ui service tho